tests/utilities/helper.py

```diff
@@ -8,7 +8,7 @@
 import string
 
 DEVOPS_CLI_TEST_ORGANIZATION = 'https://AzureDevOpsCliTest.visualstudio.com'
```

Comment:
every one developing will not have permission on this org, so I think we should make this also configurable (pass target org and pat token to runtests.ps1) so that everyone can run tests in live mode against their own org.
I know our tests are not their right now that it can run against any collection (because some of them expect some setup)
but we should try to make our tests org agnostic in long run

Reply:
This org is not forced on all the tests. External contributors can define the org in their test and run the test and get a recording. Some with the test repo access will only need to rerun the test and update the recording with our org. 

Not considering this comment for this PR. You can raise a issue to track that.